app/assets/javascripts/oxalis/model/sagas/volumetracing_saga.js

```diff
@@ -23,8 +23,10 @@ export function* updateIsosurface(): Generator<*, *, *> {
 
   if (shouldDisplayIsosurface) {
     activeCellIdMaybe.map(activeCellId =>
-      // $FlowFixMe
-      app.oxalis.sceneController.renderVolumeIsosurface(activeCellId),
+      // importing SceneController breaks webpack (circular dependency)
+      // TODO fix later
```

Comment:
Can you open an issue for that? It's probably not going to bite us anytime soon, but for the sake of tracking it, I think, it would be sensible to do.